app/Filament/Resources/GameResource/RelationManagers/AchievementsRelationManager.php

```diff
@@ -215,10 +250,44 @@ public function table(Table $table): Table
                                 $record->type = null;
                                 $record->save();
                             });
+
+                            Notification::make()
+                                ->title('Success')
+                                ->body('Successfully removed type from selected achievements.')
+                                ->success()
+                                ->send();
                         }),
                 ])
                     ->label('Bulk set type')
                     ->visible(fn (): bool => $user->can('updateField', [Achievement::class, null, 'type'])),
+
+                Tables\Actions\BulkAction::make('add-credit')
+                    ->label('Bulk add credit')
```

Comment:
Is this supposed to have three vertical dots too?

![image](https://github.com/user-attachments/assets/b17cc17c-9039-4c8b-b080-c2ce077a74af)

Reply:
This one shouldn't, as it doesn't open a dropdown menu.